cpp/.clang-format

```diff
@@ -0,0 +1,7 @@
+---
+Language: Cpp
```

Comment:
Did you want to check commit this file?
Is this a local config?

Reply:
Yes, helps with formatting